### ISSUE TITLE
Null Reference Exception since version 2.0.0.0 in SectionMapper

### ISSUE THREAD

**Issue by danielburkhart-johndeere** [ADAPT/ISOv4Plugin#26]
In [SectionMapper.cs](https://github.com/ADAPT/ISOv4Plugin/blob/develop/ISOv4Plugin/Mappers/LoggedDataMappers/Import/SectionMapper.cs), line 52, I get a NullReferenceException sind version 2.0.0.0 (Parent is Null):
```
if (config.DeviceElementId == adaptDeviceElement.ParentDeviceId)
{
   //The configuration references the parent ISO element
   depth = hierarchy.Parent.Depth;
   order = hierarchy.Parent.Order;
}
```

**Comment by knelson-farmbeltnorth** [comment]
Daniel, if you want to fix this and submit a pull request, please feel free to do so.   Otherwise we'll grab this before the next release.     With additional users of the ISO Plugin and more variations of data that we didn't have during testing, I suspect we'll find some additional things to put into a 2.0.1 patch release.

**Comment by knelson-farmbeltnorth** [comment]
@danielburkhart-johndeere, can you recommend a fix or supply some impacted data that I can look at?  I haven't found any data that will cause the error, and I'm reluctant to simply put a null check there without understanding the data condition that is causing this.

**Comment by danielburkhart-johndeere** [comment]
Issue solved with PR #57
